rustls/src/webpki/verify.rs

```diff
@@ -151,10 +181,31 @@ impl ServerCertVerifier for WebPkiServerVerifier {
     ) -> Result<ServerCertVerified, Error> {
         let cert = ParsedCertificate::try_from(end_entity)?;
 
-        verify_server_cert_signed_by_trust_anchor(
+        let crls = borrow_crls(&self.crls);
+        let revocation = if crls.is_empty() {
+            None
+        } else {
+            // Note: unwrap here is safe because RevocationOptionsBuilder only errors when given
+            //       empty CRLs.
+            let mut builder = webpki::RevocationOptionsBuilder::new(&crls)
```

Comment:
The `borrow_crls(&self.crls)` -> `RevocationOptionsBuilder::new(&crls).unwrap()` dance looks pretty convoluted... is this what we ended up on for the client? Maybe webpki should expose some API to make this more streamlined?

Reply:
I have a couple branches under way that should make this more ergonomic. Since it will require some song/dance to get webpki fixed up and then a new alpha used here maybe we could look at that separately from this PR? WDYT?

Reply:
> I have a couple branches under way that should make this more ergonomic.

Webpki: https://github.com/rustls/webpki/pull/203
Rustls: https://github.com/rustls/rustls/pull/1552